content/attacks/posts/2024-01-31-Abracadabra-Money.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2024-01-30
+target-entities: Abracadabra Money
+entity-types:
+  - DeFi
+  - Lending Platform
+attack-types:
+  - Smart Contract Exploit
+  - Flash Loan Attack
+title: "Abracadabra Money Suffers $6.5 Million Loss Due to Smart Contract Exploit"
+loss: 6500000
+---
+
+## Summary
+
+Abracadabra Money, a prominent leverage and lending platform in the DeFi space, was exploited on January 30, 2024, due to a smart contract vulnerability on the Ethereum Mainnet. This exploit led to the unauthorized borrowing and subsequent theft of assets, totaling over $6.5 million, including 1800 ETH and 2.2 million MIM tokens.
+
+## Attackers
+
+The attackers behind the Abracadabra Money hack remain unidentified.
+
+Hacker ETH wallets:
+- [0x87F585809Ce79aE39A5fa0C7C96d0d159eb678C9](https://etherscan.io/address/0x87F585809Ce79aE39A5fa0C7C96d0d159eb678C9)
+- [0x40d5FFA20fC0dF6bE4D9991938dAa54E6919c714](https://etherscan.io/address/0x40d5FFA20fC0dF6bE4D9991938dAa54E6919c714)
+- [0xbD12D6054827ae3fc6D23B1aCf47736691b52Fd3](https://etherscan.io/address/0xbD12D6054827ae3fc6D23B1aCf47736691b52Fd3)
+
+## Losses
+
+Abracadabra Money lost approximately $6.5 million:
+- 1800 ETH
+- 2,200,000 MIM 
+
+## Timeline
+
+- **January 30, 2024, 10:14:35 AM +UTC:** The [first malicious transaction](https://etherscan.io/tx/0x26a83db7e28838dd9fee6fb7314ae58dcc6aee9a20bf224c386ff5e80f7e4cf2) occurred.
+- **January 30, 2024, 10:26:59 AM +UTC:** A [second malicious transaction](https://etherscan.io/tx/0xdb4616b89ad82062787a4e924d520639791302476484b9a6eca5126f79b6d877) occurred.
+- **January 30, 2024, 11:00 AM +UTC:** MIM token was [deppeged](https://twitter.com/PeckShieldAlert/status/1752287909917659356/photo/1).
+- **January 30, 2024, 01:35 PM +UTC:** BlockSec has [detected](https://twitter.com/peckshield/status/1752279373779194011) a hack.
```

Comment:
BlockSec and Peckshield are different security companies. Pay attention to such details please.